src/libLookup/Lookup.cpp

```diff
@@ -3116,6 +3116,16 @@ void Lookup::SendTxnPacketToNodes(uint32_t numShards) {
     // return;
   }
 
+  // allow receving nodes to be ready with latest DS block ( Only** for first
+  // txn epoch of every ds epoch )
+  if ((m_mediator.m_currentEpochNum % NUM_FINAL_BLOCK_PER_POW == 1)) {
```

Comment:
This check may have problem, for epoch 1 is equal to 1 and if it is larger than 1, then should check (m_mediator.m_currentEpochNum % NUM_FINAL_BLOCK_PER_POW == 0)